README.md

```diff
@@ -32,8 +32,7 @@ To build the security-kibana plugin from source follow these instructions:
 │   └── plugins
 │       └── opendistro_security
 ```
-* run the build script to build the plugin inside `kibana/plugins/opendistro_security` directory
-`node build_tools/build_plugin.js`
+* run `yarn build` to build the plugin inside `kibana/plugins/opendistro_security` directory
```

Comment:
Just curious what changes made it work now.

Reply:
Kibana 7.10 fix the plugin helper, the build cmd is updated in https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/pull/627.